src/docs/product/error-monitoring/breadcrumbs/index.mdx

```diff
@@ -1,55 +1,55 @@
 ---
 title: Using Breadcrumbs
```

Comment:
breadcrumbs should probably not be it's own page but within the issue details page now? if not it should at least be below issue states and triage